src/components/AppServicesEmptyState/AppServicesEmptyState.css

```diff
@@ -0,0 +1,3 @@
+.pf-c-empty-state.pf-m-xl {
+  --pf-c-empty-state__content--MaxWidth: 624px;
```

Comment:
This should be checked against the smaller breakpoints. There is a button on the toolbar in Storybook to try different layouts

Reply:
This is existing css in kas-ui. This was added for some reason by someone.